abasnfarah/Tennis-Self-Serve
# Adds styling and updates home page

## Describe your changes

To test locally do the following:
1. Add clerk private keys from discord to `.env` and `apps/expo/app.config.ts` files
2. Then rerun the following:
```bash
pnpm i

pnpm db-push

pnpm dev
```

To see on phone first run `pnpm dev` in the `/apps/expo` directory to get the QR code. Then close the server and re run `pnpm dev` in the root directory. 

### Changes
1. Adds the styling and logo to the home page.
2. Sets up the expo camera config for future PR's with a record function.
3. Changes colorscheme to Raeda color scheme.

#### Issues
closes #16 

## Checklist before requesting a review
